merfi/tests/test_iso.py

```diff
@@ -0,0 +1,24 @@
+import os
+import pytest
+import subprocess
+from merfi.iso import Iso
+
+class TestIso(object):
+
+    def setup(self):
+        self.iso = Iso([])
+
+    def test_sha256sum(self, tmpdir):
+        f = tmpdir.join('test.iso')
+        f.write('ISOCONTENTS')
+        self.iso.output = str(f)
+        self.iso.make_sha256sum()
+        checksum_file = self.iso.output + '.SHA256SUM'
+        with open(checksum_file, 'r') as shaf:
+            assert shaf.read() == "d8d322f6864229f8c9ef1b0845dd9e182c563c508fec30618fdb9b57c70a0147  test.iso\n"
+            # ... and validate checksum_file's syntax with sha256sum.
+            # The reason we shell out to sha256sum here is that it
+            # functionally validates what a user would do with this ISO's
+            # checksum file: a user would run `sha256sum -c` on it.
+            os.chdir(os.path.dirname(checksum_file))
+            assert subprocess.call(['sha256sum', '-c', checksum_file]) == 0
```

Comment:
this adds a system dependency and it means I wouldn't be able to run it locally.

Would the bug that got caught be detected with just Python? Or that was just not possible?